addon/dialog/dialog.js

```diff
@@ -82,7 +82,9 @@
         if (e.keyCode == 13) callback(inp.value, e);
       });
 
-      if (options.closeOnBlur !== false) CodeMirror.on(inp, "blur", close);
+      if (options.closeOnBlur !== false) CodeMirror.on(dialog, "focusout", function (evt) {
+        if (evt.relatedTarget !== null) close();
+      });
     } else if (button = dialog.getElementsByTagName("button")[0]) {
```

Comment:
While you are at it, can you correct this assignment to make it a comparison:
```javascript
} else if (button = dialog.getElementsByTagName("button")[0]) {
```
to 
```javascript
} else if (button == dialog.getElementsByTagName("button")[0]) {
```

Reply:
No, that would not be a good idea, since it would break the code.

Reply:
Great ! I see, thanks.